pkg/sense_hat_adapter.py

```diff
@@ -121,12 +121,21 @@ def __init__(self, adapter):
                 "color",
                 {
                     '@type': 'ColorProperty',
-                    'label': "Color",
+                    'label': "Light Color",
                     'type': 'string',
                     'readOnly': False
                 },
                 '#ffffff')
-
+            self.properties['lightOn'] = SenseHatProperty(
```

Comment:
I think "on" is better here as there is only one "switch"

Reply:
done